### PR TITLE
Update default ice climatology in ./ush/global_cycle.sh.

### LINKED ISSUE #661: Update default ice climatology in ./ush/global_cycle.sh
The global-workflow group would like the following change to `./ush/global_cycle.sh`.

FNAISC=${FNAISC:-"$FIX_AM/CFSR.SEAICE.1982.2012.monthly.clim.grb"}
with
FNAISC=${FNAISC:-"$FIX_AM/IMS-NIC.blended.ice.monthly.clim.grb"}

No source code changes were requested.

See https://github.com/NOAA-EMC/global-workflow/issues/886

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
## DESCRIPTION OF CHANGES: 
Update default ice climatology in ./ush/global_cycle.sh

## TESTS CONDUCTED: 
Because GFS OPS currently uses the old ice data, the global_cycle consistency test was adjusted to use it. Therefore, the consistency test passed. At some point a new test may be warranted.

## DEPENDENCIES:
None.

## DOCUMENTATION:
N/A

## ISSUE: 
Fixes #661.

### CODE REVIEW

**Comment by GeorgeGayno-NOAA** [comment]
@KateFriedman-NOAA, you may want to run your own cycled test.

File: reg_tests/global_cycle/C768.fv3gfs.sh [hunk 1]
```diff
@@ -33,6 +33,7 @@ export LONB=3072
 export LATB=1536
 
 export FIXgsm=$BASE_GSM/fix/fix_am
+export FNAISC=$FIXgsm/CFSR.SEAICE.1982.2012.monthly.clim.grb
```
Comment (KateFriedman-NOAA) [inline reg_tests/global_cycle/C768.fv3gfs.sh:36]: Should this be the `IMS-NIC.blended.ice.monthly.clim.grb` or are you specifically using the prior one here?

**Comment by KateFriedman-NOAA** [comment]
@GeorgeGayno-NOAA Is this for GFSv16 or GFSv17?

File: reg_tests/global_cycle/C768.fv3gfs.sh (hunk 1 above)
Comment (GeorgeGayno-NOAA) [reply reg_tests/global_cycle/C768.fv3gfs.sh:36]: I am intentionally using the old data for the regression test. OPS still uses old data, correct?

**Comment by GeorgeGayno-NOAA** [comment]
> @GeorgeGayno-NOAA Is this for GFSv16 or GFSv17?

@yangfanglin, do you know?

**Comment by yangfanglin** [comment]
The new ice data should be used for all future ufs-weather-model applications, not for any GFS.v16 based operations.

File: reg_tests/global_cycle/C768.fv3gfs.sh (hunk 1 above)
Comment (KateFriedman-NOAA) [reply reg_tests/global_cycle/C768.fv3gfs.sh:36]: Ah, gotcha. Yes the GFSv16 is using the older (`CFSR.SEAICE.1982.2012.monthly.clim.grb`) file.

**Comment by KateFriedman-NOAA** [comment]
> The new ice data should be used for all future ufs-weather-model applications, not for any GFS.v16 based operations.

Ok, thanks for confirming @yangfanglin ! Since this is for GFSv17+ I'm pulling @WalterKolczynski-NOAA in. He's the primary one managing the global-workflow `develop` branch at the moment.

**Review by KateFriedman-NOAA — APPROVED** [review]
Change looks good to me but I'm still transitioning out of GFSv16 (ops) work back to development work so I'm not available to test within global-workflow `develop` right now.

**Comment by WalterKolczynski-NOAA** [comment]
Looks like this should go into global-workflow at the same time as the other seaice change in https://github.com/NOAA-EMC/global-workflow/pull/902? If so, once this is merged, that PR should be updated to check out the new UFS_UTILS version.

**Review by JeffBeck-NOAA — APPROVED** [review]

**Comment by GeorgeGayno-NOAA** [comment]
Retested the branch at 8a278b9 after several merges. The `global_cycle` regression tests passed on Hera, Jet, Orion and WCOSS2.

**Pull request merged**